lib/ui/screens/contact_list_screen.dart

```diff
@@ -64,7 +65,10 @@ class _ContactListState extends State<ContactList> {
     @override
     Widget build(BuildContext context) {
         _pushAdd(){
```

Comment:
function should be, change move it to a dedicated function (outside this build function):

```suggestion
        _goToAddNewContact(BuildContext context) {
    Navigator.push(
        context, MaterialPageRoute(builder: (context) => NewContact()));
  }
```